modules/host/default.nix

```diff
@@ -4,18 +4,41 @@
   self,
   microvm,
   netvm,
-}: {lib, ...}: {
-  imports = [
-    (import ./minimal.nix)
+}: {
+  lib,
+  config,
+  pkgs,
+  ...
+} @ args: let
+  cfg = config.ghaf.host.minification;
+in
+  with lib; {
+    imports = [microvm.nixosModules.host];
 
-    microvm.nixosModules.host
+    options.ghaf.host.minification = with lib; {
+      reduceProfile = mkEnableOption "reduced profile";
+      disableNetwork = mkEnableOption "disableing of network";
+      disableGetty = mkEnableOption "disabling of getty";
+      removeNix = mkEnableOption "nix tooling";
+      disableGuestVms = mkEnableOption "guest vms";
+    };
```

Comment:
I meant adding `description`s to `mkEnableOption`s. Seems you chose to ignore the request and resolved those instead. Ok, postponed for later.